code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1544,6 +1544,11 @@
 						var/obj/container = src.loc
 						container.mob_flip_inside(src)
 
+#if ASS_JAM
+					if (ismob(src.loc))
+						var/mob/container = src.loc
+						container.mob_flip_inside(src)
```

Comment:
```suggestion
					if (ismob(container))
						var/mob/mob_container = container
						mob_container.mob_flip_inside(src)
```
Note that this would also apply for other... ways... of having a mob inside another mob